tests/dns/task-7018-dns-ips-stream-rule/README.md

```diff
@@ -0,0 +1,28 @@
+# Test Description
+
+Test earlier alert matches after trigger raw stream reassembly when there's a
+completed DNS TCP transaction.
+
+## PCAP
+
+dns-tcp-multi.pcap, crafted for this test, shared in the Redmine ticket.
+
+The capture shows three request-response DNS transactions:
+Query 1: suricata.io
+Query 2: oisf.net
+Query 3: suricata.org
+
+## Behavior
+
+We match those against a single payload rule without any DNS keywords,
+and inspecting content `suricata|02|`. Ideally, the expectation is to have 2 alerts,
+for the portion of the stream associated with Query 1 - that's because on the
+wire we observe that for Query three the content is `suricata|03|`.
+
+For IPS mode, as a larger portion of the stream buffer is kept available, and as
+it still contains the matching bytes, we'll see more alerts triggered.
+
```

Comment:
I think we should also call-out the undesirable behavior here.  Our content is looking for `suricata|02|`, however we are logging DNS `alerts` with transaction data for `oisf.net`.

Maybe also test for that undesirable behavior as well.